src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,47 @@
+@use '../_variables/variables' as *;
+@use '../mixins/mixins.scss' as *;
+
+.nav {
+  &__list {
+    display: flex;
+    margin-top: 0;
+    margin-bottom: 0;
+    padding-left: 0;
+  }
+
+  &__item:not(:last-child) {
+    margin-right: 20px;
+  }
+
+  &__link {
+    display: block;
+    text-decoration: none;
+    color: inherit;
+    font-size: 12px;
+    text-transform: uppercase;
+    line-height: 60px;
+    font-weight: 500;
+
+    @include transition-options(color);
+
+    &:is(:hover, :focus) {
+      color: $ColorBlueAccent;
+    }
+
+    &.is-active {
+      position: relative;
+      color: $ColorBlueAccent;
+
+      &::after {
+        content: '';
+        position: absolute;
+        bottom: 0;
+        left: 0;
+        width: 100%;
+        height: 4px;
+        border-radius: 8px;
+        background-color: $ColorBlueAccent;
```

Comment:
The variable `$ColorBlueAccent` should be written in lowercase according to the code style rules.